frontend/src/components/survey-handler/survey-handler.test.tsx

```diff
@@ -49,6 +49,22 @@ describe('SurveyHandler', () => {
       expect(screen.getByText('Question 1')).toBeInTheDocument();
     });
 
+    it('should not render an image when question does not have an image url', () => {
+      const mockSurvey = JSON.parse(JSON.stringify(surveyDefault)) as Survey;
+      mockSurvey.current_answers_survey.questions[0].image_url = '';
```

Comment:
What's happen when the attribute is null?

Reply:
Same behaviour of an empty string and `undefined`, don't show the image.

Reply:
> Same behaviour of an empty string and `undefined`, don't show the image.

I belive that nescessary add a test for this.